hash_practice/exercises.py

```diff
@@ -2,28 +2,123 @@
 def grouped_anagrams(strings):

```

Comment:
✨ I believe time complexity is just O(nk), n being the length of `strings` and coming from your for loop on line 13, and the call to `create_dict` inside it having time complexity k where k is the longest word in `strings` . Space complexity would echo this, with your `dict_list` being of length n and and each dictionary inside it having up to k elements.